docs/Configuration.md

```diff
@@ -206,9 +206,39 @@ Configuration of processes has few common keys:
 If configuration does not contain `checkpointConfig`, `checkpointInterval` and process does not contain `checkpointInterval` in its properties then checkpoints are not enabled for process.
 
 The rest of model configuration depends on your needs - all the properties defined here will be passed to ```ProcessConfigCreator``` as explained in [API](API.md) documentation.
+                                                   
+###Configuration of component providers
 
-###Configuration of services
-In model configuration you can also define some attributes of services. These include:
+```
+  componentProviders {
+    sqlHsql {
+      providerType: sql
+      jdbcUrl: jdbc:hsql...//
```

Comment:
after taking a look to examples in code, it looks like those properties should be nested in `config`. Maybe we should make it that it won't be nested? WDYT?

Reply:
it's a bit tricky, but OK - I did it :+1: